### PR TITLE
enhacement/detekt newline variablenaming

### PR DESCRIPTION
**Pull request by sierisimo**
- Fixing Detekt issues for VairableName and NewLine at end of files

### CODE REVIEW

File: Android/Nothing/fastlane/report.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
+
```
Comment (ChubyAvodroc) [inline Android/Nothing/fastlane/report.xml:4]: Remove this empty lines, please

File: Android/Nothing/fastlane/report.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<testsuites>
+  <testsuite name="fastlane.lanes">
+    
+    
+    
+      
+      <testcase classname="fastlane.lanes" name="0: default_platform" time="0.000562">
+        
+      </testcase>
+
```
Comment (ChubyAvodroc) [inline Android/Nothing/fastlane/report.xml:11]: Wild empty line

File: Android/Nothing/build.gradle [hunk 3]
```diff
@@ -9,7 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.0-alpha13'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.3.2'
+        classpath 'com.google.gms:google-services:4.3.4'
```
Comment (ChubyAvodroc) [inline Android/Nothing/build.gradle:11]: Can we move all versions inside ext block?

File: Android/Nothing/app/quality.gradle [hunk 4]
```diff
@@ -0,0 +1,15 @@
+detekt {
+    toolVersion = "1.14.1"
+    config = files("$rootProject.projectDir/config/detekt/detekt.yml")
+
+    reports {
+        xml {
+            enabled = true
+//            destination = file("$project.reporting.baseDir/detekt.xml")
```
Comment (ChubyAvodroc) [inline Android/Nothing/app/quality.gradle:8]: Is this going to be used in a near future? if not then move it to alv

File: Android/Nothing/app/quality.gradle [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        }
+        html {
+            enabled = true
+//            destination = file("$project.reporting.baseDir/detekt.html")
```
Comment (ChubyAvodroc) [inline Android/Nothing/app/quality.gradle:12]: Same here

**Review by ChubyAvodroc — COMMENTED** [review]
🙃🙃🙃🙃🙃

File: Android/Nothing/fastlane/report.xml (hunk 1 above)
Comment (sierisimo) [reply Android/Nothing/fastlane/report.xml:4]: This file shouldn't be here. I'll deleted and add it to gitignore on a different PR

File: Android/Nothing/fastlane/report.xml (hunk 2 above)
Comment (sierisimo) [reply Android/Nothing/fastlane/report.xml:11]: Same, file Is generated, no need for it at repo

File: Android/Nothing/build.gradle (hunk 3 above)
Comment (sierisimo) [reply Android/Nothing/build.gradle:11]: Better implementation for dependencies will be in a different PR

File: Android/Nothing/app/quality.gradle (hunk 4 above)
Comment (sierisimo) [reply Android/Nothing/app/quality.gradle:8]: Can be used on tools like bitrise, sonarqube or similar.

File: Android/Nothing/app/quality.gradle (hunk 5 above)
Comment (sierisimo) [reply Android/Nothing/app/quality.gradle:12]: This is currently used for local development. The report tells you what will break the build due to lack of quality

File: Android/Nothing/build.gradle (hunk 3 above)
Comment (gorrotowi) [reply Android/Nothing/build.gradle:11]: Maybe in a buildSrc. Yeah! I know the exit!

**Review by gorrotowi — APPROVED** [review]
LGTM 👍  ALV!

**Pull request merged**